bundle/language_en.properties

```diff
@@ -305,6 +305,7 @@ angal.billbrowser.deletetheselectedbill.msg
 angal.billbrowser.editbill                                                                             = Edit Bill
 angal.billbrowser.editbill.btn                                                                         = Edit Bill
 angal.billbrowser.editbill.btn.key                                                                     = E
+angal.billbrowser.from                                                                                 = From
```

Comment:
why to add billbrowser entries if we the changes concern pharmacy? also, "from" and "to" should be available under "common" keys